From rpjday@crashcourse.ca  Fri Jul 10 15:28:08 2009
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Sun, 5 Jul 2009 05:19:29 -0400 (EDT)
Subject: Staging: vt6655: Remove Makefile refs to EXPORT_SYMTAB
To: Greg KH <greg@kroah.com>
Message-ID: <alpine.LFD.2.00.0907050517450.15205@localhost>

Given that EXPORT_SYMTAB was removed long ago, remove the Makefile
references to it.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/vt6655/Makefile     |    2 +-
 drivers/staging/vt6655/Makefile.arm |    2 +-
 drivers/staging/vt6655/Makefile.x86 |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/vt6655/Makefile
+++ b/drivers/staging/vt6655/Makefile
@@ -1,5 +1,5 @@
 # TODO: all of these should be removed
-EXTRA_CFLAGS += -DLINUX -D__KERNEL__ -DMODULE  -DEXPORT_SYMTAB -D__NO_VERSION__
+EXTRA_CFLAGS += -DLINUX -D__KERNEL__ -DMODULE -D__NO_VERSION__
 EXTRA_CFLAGS += -DHOSTAP
 
 vt6655-y +=	device_main.o \
--- a/drivers/staging/vt6655/Makefile.arm
+++ b/drivers/staging/vt6655/Makefile.arm
@@ -118,7 +118,7 @@ OBJS :=	device_main.o card.o mac.o baseb
 CC := /usr/local/bin/arm-linux-gcc
 LD := /usr/local/bin/arm-linux-ld
 
-CFLAGS += -Wall -DLINUX -D__KERNEL__ -DMODULE  -DEXPORT_SYMTAB -D__NO_VERSION__ -O2 -pipe
+CFLAGS += -Wall -DLINUX -D__KERNEL__ -DMODULE -D__NO_VERSION__ -O2 -pipe
 #CFLAGS += -Wstrict-prototypes -fomit-frame-pointer
 COPTS+= -march=armv4 -fno-strict-aliasing -fno-common
 #COPTS+= -mapcs-32 -mtune=xscale  -mshort-load-bytes -msoft-float -mfp=2
--- a/drivers/staging/vt6655/Makefile.x86
+++ b/drivers/staging/vt6655/Makefile.x86
@@ -128,7 +128,7 @@ test_cc = $(shell which $(cc) > /dev/nul
 CC := $(foreach cc, $(CC), $(test_cc))
 CC := $(firstword $(CC))
 
-CFLAGS += -Wall -DLINUX -D__KERNEL__ -DMODULE  -DEXPORT_SYMTAB -D__NO_VERSION__ -O2 -pipe
+CFLAGS += -Wall -DLINUX -D__KERNEL__ -DMODULE -D__NO_VERSION__ -O2 -pipe
 CFLAGS += -I$(KSRC)/include -Wstrict-prototypes -fomit-frame-pointer
 CFLAGS += $(shell [ -f $(KSRC)/include/linux/modversions.h ] && \
             echo "-DMODVERSIONS -include $(KSRC)/include/linux/modversions.h")
